backend/README.md

```diff
@@ -1,3 +1,25 @@
 # Code-First
 
-> This is where you'll write your code-first implementation of the User Story from DDDForum. You can [see the assignment page for more details](https://www.essentialist.dev/products/the-software-essentialist/categories/2153149734/posts/2168948146).
\ No newline at end of file
+## Project Setup
+
+```sh
+npm install
```

Comment:
It should be `npm ci` to respect the package-lock.json - some info on it https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci.